com.unity.uiwidgets/Runtime/cupertino/date_picker.cs

```diff
@@ -1487,45 +1493,50 @@ class _CupertinoTimerPickerState : State<CupertinoTimerPicker> {
-                    ), 
-                offAxisFraction: offAxisFraction,
-                itemExtent: CupertinoDatePickerUtils._kItemExtent,
-                backgroundColor: widget.backgroundColor,
-                squeeze: CupertinoDatePickerUtils._kSqueeze,
-                looping: true,
-                onSelectedItemChanged: (int index) =>{ 
-                    setState(()=> { 
-                        selectedSecond = index * widget.secondInterval; 
-                        widget.onTimerDurationChanged(
-                            new TimeSpan(
-                                hours: selectedHour == 0 ? 0 : selectedHour, 
-                                minutes: selectedMinute, 
-                                seconds: selectedSecond)); 
-                    });
-              }, 
-                children: CupertinoDatePickerUtils.listGenerate((int)60 / widget.secondInterval, (int index)=> { 
-                    int second = index * widget.secondInterval;
-                    string semanticsLabel = textDirectionFactor == 1 
-                        ? localizations.timerPickerSecond(second) + localizations.timerPickerSecondLabel(second) 
-                        : localizations.timerPickerSecondLabel(second) + localizations.timerPickerSecond(second);
-
-                    return _buildPickerNumberLabel(localizations.timerPickerSecond(second), additionalPadding);
-                })
-                ); 
-        }
-        Widget _buildSecondColumn(EdgeInsetsDirectional additionalPadding) {
+              scrollController: new FixedExtentScrollController(
+                initialItem: (int) selectedSecond / widget.secondInterval
+              ),
+              offAxisFraction: offAxisFraction,
+              itemExtent: CupertinoDatePickerUtils._kItemExtent,
+              backgroundColor: widget.backgroundColor,
+              squeeze: CupertinoDatePickerUtils._kSqueeze,
+              looping: true,
+              onSelectedItemChanged: (int index)=> {
+                setState(() => {
+                  selectedSecond = index * widget.secondInterval;
+                  widget.onTimerDurationChanged(
+                    new TimeSpan(
+                      hours: selectedHour == 0 ? 0 : selectedHour,
+                      minutes: selectedMinute,
+                      seconds: selectedSecond));
+                });
+              },
+              children: CupertinoDatePickerUtils.listGenerate((int) (60 / widget.secondInterval), (int index)=> {
+                 int second = index * widget.secondInterval;
+
+                 string semanticsLabel = textDirectionFactor == 1
+                  ? localizations.timerPickerSecond(second) + localizations.timerPickerSecondLabel(second)
+                  : localizations.timerPickerSecondLabel(second) + localizations.timerPickerSecond(second);
+
+                return _buildPickerNumberLabel(localizations.timerPickerSecond(second), additionalPadding);
+                
```

Comment:
remove this empty line (you can trycode cleanup with UIWidgets editor configurations)